plugins/hydra_nevergrad_sweeper/setup.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as fh:
+    LONG_DESC = fh.read()
+    setup(
+        name="hydra_nevergrad_sweeper",
+        version="0.1.0",
+        author="Jeremy Rapin",
+        author_email="jrapin@fb.com",
+        description="Hydra Nevergrad Sweeper plugin with basic features",
```

Comment:
No need to be apologetic, and this plugin can grow to cover additional features.

```suggestion
        description="Hydra Nevergrad Sweeper plugin",
```